### DISCUSSION TITLE
Improve my Mail Deliverability

### DISCUSSION THREAD

**Discussion by PeterGDSoft** [docker-mailserver/docker-mailserver#4214] [Q&A] [answered]
Hi I have started the mail server, everything is working nicely I have a score of 10/10 on the mail test pages, I have all the DNS records well set (PTR, DMARC, DKIM, SPF)

However, emails from my server go into SPAM if I send them to my gmail mailbox (I care about gmail the most)
Do you know what I can do to improve the deliverability of my emails? Mainly with gmail in mind, but overall so that they don't go into SPAM everywhere

I am not on blacklists, the server is hosted on OVH

**Comment by polarathene** [comment, marked as answer]
> Do you know what I can do to improve the deliverability of my emails?

You need to look into sender reputation, this is often an issue faced with self-hosting your own mail server AFAIK if you do not relay mail through a service (_such as SendGrid_) that has established a trusted reputation with the big providers like Gmail.

Not much DMS can do here AFAIK. Users that receive the mail can usually flag the mail as "not spam" on their side which should help the service like Gmail recognize the mail server is legitimate and that may eventually raise the sender reputation higher to not land in spam/junk folder on new inboxes you deliver to for that service.

**Comment by bocan** [comment]
Just my opinion, but running anything on OVH is a recipe to wind up on a blacklist. I think half the spam I receive comes from there. Once, several years back, I went to the trouble of finding our what ALL their entire network ranges were and blocked them off from our corporate servers along with my personal one.
